src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -1206,6 +1206,61 @@ def test_aks_stop_and_start(self, resource_group, resource_group_location):
         start_cmd = 'aks start --resource-group={resource_group} --name={name}'
         self.cmd(start_cmd)
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_abort(self, resource_group, resource_group_location):
+        aks_name = self.create_random_name('cliakstest', 16)
+        self.kwargs.update({
+            'resource_group': resource_group,
+            'name': aks_name,
+            'ssh_key_value': self.generate_ssh_keys(),
+        })
+
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} --ssh-key-value={ssh_key_value} --no-wait'
+        self.cmd(create_cmd)
+
+        abort_cmd = 'aks operation-abort --resource-group={resource_group} --cluster-name={name}'
+        self.cmd(abort_cmd, checks=[self.is_empty()])
```

Comment:
Seems like this is missing a check on the ProvisioningState == `Canceled`

Reply:
Added check